docs/developer_guide/lane_pose_calculation/UpdateFrame.md

```diff
@@ -47,11 +47,11 @@ The procedure for calculating the pose in the lane coordinate system at this tim
 
 If lane matching was successful in the previous frame, do 1, otherwise do 2
 
-1.[Set the length of the horizontal bar to 2.0 and calculate the pose in the lane coordinate system](https://github.com/tier4/scenario_simulator_v2/blob/5f19d39ef29243396f26225976975f0c 27914c12/simulation/behavior_tree_plugin/src/pedestrian/pedestrian_action_node.cpp#L72-L77)  
-2.[Calculate the pose in the lane coordinate system considering the size of the BoundingBox of the Entity](https://github.com/tier4/scenario_simulator_v2/blob/5f19d39ef29243396f26225976975f0 c27914c12/simulation/behavior_tree_plugin/src/pedestrian/pedestrian_action_node.cpp#L79-L83)  
+1.[Set the length of the horizontal bar to 2.0 and calculate the pose in the lane coordinate system](https://github.com/tier4/scenario_simulator_v2/blob/5f19d39ef29243396f26225976975f0c27914c12/simulation/behavior_tree_plugin/src/pedestrian/pedestrian_action_node.cpp#L72-L77)  
+2.[Calculate the pose in the lane coordinate system considering the size of the BoundingBox of the Entity](https://github.com/tier4/scenario_simulator_v2/blob/5f19d39ef29243396f26225976975f0c27914c12/simulation/behavior_tree_plugin/src/pedestrian/pedestrian_action_node.cpp#L79-L83)  
```

Comment:
The links and descriptions are old ones.
Please match the documentation with the latest code.

Reply:
Fixed in https://github.com/tier4/scenario_simulator_v2/pull/1194/commits/17b5d94e0c85728d58287556808cff182959389e and https://github.com/tier4/scenario_simulator_v2/pull/1194/commits/e02b39cdd5d58dedbea8788c8d42154cf1dac2c5